<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPurchasesOrderView</class>
 <widget class="QWidget" name="AddPurchasesOrderView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add purchases order</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="addProductPushButton">
        <property name="toolTip">
         <string>Add new product</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images.qrc">
          <normaloff>:/images/manufacturers/add_manufacturer.png</normaloff>:/images/manufacturers/add_manufacturer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editProductPushButton">
        <property name="toolTip">
         <string>Edit product</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images.qrc">
          <normaloff>:/images/manufacturers/edit_manufacturer.png</normaloff>:/images/manufacturers/edit_manufacturer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeProductPushButton">
        <property name="toolTip">
         <string>Remove product</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../images.qrc">
          <normaloff>:/images/manufacturers/remove_manufacturer.png</normaloff>:/images/manufacturers/remove_manufacturer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="6">
       <widget class="QTableWidget" name="itemsTableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Product Code</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Product Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Quantity</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Cargo</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QDoubleSpinBox" name="cargoDoubleSpinBox">
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QLineEdit" name="totalValueLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okPushButton">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelPushButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Receipt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Payment Condition</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="8">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="receiptLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="paymentConditionLineEdit">
        <property name="text">
         <string>1</string>
        </property>
        <property name="maxLength">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Parcel(s)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>$</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Payment Form</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="8">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Observation</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="6">
       <widget class="QPlainTextEdit" name="observationPlainTextEdit"/>
      </item>
      <item row="10" column="6">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>156</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <widget class="QComboBox" name="providerComboBox"/>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="paymentFormComboBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Provider</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
      <item row="4" column="5">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="valuePaidDoubleSpinBox">
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Paid Value</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
